app/src/main/java/fr/neamar/kiss/broadcast/InstallShortcutHandler.java

```diff
@@ -61,7 +61,7 @@ public void onReceive(Context context, Intent data) {
             }
         }
                 
-        dh.getShortcutProvider().addShortcut(pojo);
+        dh.addShortcut(pojo);
```

Comment:
Hum, I don't understand why we can't keep this on the Shortcut Provider?

Seems better to put it here than on the DataHandler?